src/registrar/tests/common.py

```diff
@@ -116,6 +116,31 @@ def __init__(self, admin, model=None, url=None, user=None, factory=None, client=
         self.url = url
         self.client = client
 
+    def assert_response_contains_distinct_values(self, response, expected_values):
```

Comment:
(Non-blocking): This may be over-engineering. I think we'll (except for you) forget this exists and write assertions in future tests.